firedrake/mesh.py

```diff
@@ -2010,8 +2010,10 @@ def spatial_index(self):
         # tolerance. Note that if tolerance is too small it might not actually
         # change the value!
```

Comment:
Assuming the answer to my other comment is "yes", then please update this comment to reflect changes below